@shared/api/internal.ts

```diff
@@ -1162,7 +1163,9 @@ export const editCustomNetwork = async ({
   return response;
 };
 
-export const loadSettings = (): Promise<Settings & IndexerSettings> =>
+export const loadSettings = (): Promise<
+  Settings & IndexerSettings & { assetsLists: AssetsLists }
```

Comment:
assets lists don't quite fall into the paradigm of general app settings or settings from the indexer. Maybe if this "3rd thing" category grows, we can add a separate service to deal with it